pkg/sbom/sbom_interface.go

```diff
@@ -0,0 +1,7 @@
+package sbom
```

Comment:
[Required]: See my comment above regarding "nesting" datastructures in "version" packages